### ISSUE TITLE
nodemon doesn't respect ignore pattern for directories created after starting nodemon

### ISSUE THREAD

**Issue by khoomeister** [remy/nodemon#574]
Assuming `nodemon` is in your path, this is the repro case:

```bash
mkdir -p modules/a/test
nodemon -e js -w modules --ignore modules/*/test -x "echo restart"
touch modules/a/test/no-restart-here.js

# new directory
mkdir -p modules/b/test
touch modules/b/test/this-should-make-it-restart-when-it-shouldnt.js
```

**Comment by remy** [comment]
Please try the build here #625 and feedback against the issue #625.

**Comment by remy** [comment]
(this is a good test case by the way - I'll see if I can replicate in the tests, or happily take a PR).

**Comment by remy** [comment]
#625 closes and released in `nodemon@1.4.2`
